pom.xml

```diff
@@ -103,7 +103,7 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
             <version>4.2.0</version>
-        </dependency>
+          </dependency>
```

Comment:
stray whitespace change

Reply:
That was not my change, that was Professor Conrad. He works in mysterious ways.

Reply:
lol